examples/tutorial/js/part6/Earthfile

```diff
@@ -43,8 +43,7 @@ api-docker:
     ENTRYPOINT ["node", "server.js"]
     SAVE IMAGE js-api:$tag
 
-# Run your app and api side by side
```

Comment:
This makes me wonder if we should add comments to promote `earthly docs`

Reply:
Maybe... some other day / PR. Feels like a bigger refactor to update all the examples.